docs/requirements.md

```diff
@@ -0,0 +1,61 @@
+At its heart ServiceControl and ServicePulse provide operational support for systems built on the Particular Service Platform. They do this by:
+
+- monitoring system health
+- diagnosing and treating errors
+
+In this context, system refers to NServiceBus endpoints and communication with third-party services.
+
+ServiceControl should provide the following features:
+
+- process and store the contents of the error queue
+- replay messages from the error queue
+- process and store the contents of the audit queue
+- monitor the availability of endpoints
+- generate and collect saga state changes
+- provide an API for monitoring user-defined conditions
+- provide integration hooks for developers to consume
+- search for messages
+- export failed message data for external analysis
+- report on the contents of the timeout manager
```

Comment:
Not sure if this requirement belongs here. Especially that we want to move away from the timeout manager towards the native implementations. These native implementations would be transport-specific and would require transport-aware management tooling.